ui-cypress/test-module/run.sh

```diff
@@ -24,5 +24,24 @@ ps aux | grep Xvfb
 # disable color output when running Cypress
 export NO_COLOR=1
 #export ELECTRON_EXTRA_LAUNCH_ARGS=--remote-debugging-port=9222
+
+# setup proxy environment variables
+if [ -n "${PROXY_HOST:-}" ]; then
+  if [ -n "${PROXY_HTTPS_PORT:-}" ]; then
+    export HTTP_PROXY="https://${PROXY_HOST}:${PROXY_HTTPS_PORT}"
+  elif [ -n "${PROXY_HTTP_PORT:-}" ]; then
+    export HTTP_PROXY="http://${PROXY_HOST}:${PROXY_HTTP_PORT}"
+  fi
+  if [ -n "${PROXY_CA_PATH:-}" ]; then
+    export NODE_EXTRA_CA_CERTS=${PROXY_CA_PATH}
+  fi
+  if [ -n "${PROXY_OBSERVABILITY_PORT:-}" ] && [ -n "${HTTP_PROXY:-}" ]; then
+    echo "Waiting for proxy"
+    curl --silent  --retry ${PROXY_RETRY_ATTEMPTS:-3} --retry-connrefused --retry-delay ${PROXY_RETRY_DELAY:-10} \
```

Comment:
do we need to check the return code here?
else it just passes over and will still run the test itself

Reply:
check has been added